samza-core/src/main/java/org/apache/samza/container/grouper/task/TaskNameGrouper.java

```diff
@@ -50,4 +51,8 @@
    * @return Set of containers, which contain the tasks that were passed in.
    */
   Set<ContainerModel> group(Set<TaskModel> tasks);
+
+  default Set<ContainerModel> group(Set<TaskModel> tasks, List<Integer> containersIds) {
+    return null;
```

Comment:
If an implemented class uses this default group method, does it work with returned null or it has to override with some implementation?